### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by rkalra247**

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+            type="text"
+            className="form-control form-control-lg font-weight-light mb-3"
+          />
+          <button
+            type="button"
+            className="btn btn-primary btn-lg btn-block mb-3"
+          >
+            Login to Your Account
+          </button>
+          <a href="">Forgot your password?</a>
```
Comment (songz) [inline pages/login.tsx]: could you wrap this around a `p` tag? Its better for screen readers.

Comment (rkalra247) [reply pages/login.tsx]: are you referring to <a>? wrap the p tag around <a>?

Comment (rkalra247) [reply pages/login.tsx]: > could you wrap this around a `p` tag? Its better for screen readers.

are you referring to wrapping the p tag over a tag?

Comment (Wolfy64) [reply pages/login.tsx]: Yes he does `<p><a>link</a></p>`

Comment (songz) [reply pages/login.tsx]:
```
import Link from 'next/link'
...
<Link href="/forgotPassword">
  <a>Forgot Password?</a>
</Link>
```

**Pull request merged**